src/Service/Player/Url.php

```diff
@@ -109,11 +109,11 @@ public function __toString()
      *
      * @see https://docs.theplatform.com/help/player-player-autoplay
      *
-     * @param bool $autoplay True to enable autoplay, false otherwise.
+     * @param bool $autoPlay True to enable autoPlay, false otherwise.
      */
-    public function setAutoplay(bool $autoplay)
+    public function setAutoplay(bool $autoPlay)
```

Comment:
If we are following camel case should this be setAutoPlay similar to the variable autoPlay?

Reply:
I hesitated to do that b/c that would change the public interface and be a breaking change. Plus require changing all the callers. I wanted to keep this low risk and quick.

Reply:
Sounds good to me